### ISSUE TITLE
Remove 'cb triggered' console log

### ISSUE THREAD

**Issue by oliver-youdooh** [wavezync/vue3-google-signin#53]
Hi there,

Any chance of removing this debugging statement? Currently this is outputting in our production env. https://github.com/wavezync/vue3-google-signin/blob/2bdfffd29919e4df6cf857f3223c86673304d9fd/src/composables/useOneTap.ts#L222

I find it would be useful only in a development environment. Perhaps we can make this a development environment output instead?

```
if (process.env.NODE_ENV === "development") {
    console.log("cb triggered"); 
}
```

Happy to open a pull request if you like.

**Comment by kasvith** [comment]
sure, please open a PR :)

**Comment by oliver-youdooh** [comment]
@kasvith thanks! Opened PR #54

**Comment by kasvith** [comment]
Closed by #54
